test/test_basic_usage.py

```diff
@@ -0,0 +1,23 @@
+import os
+import shutil
+import sys
+import unittest
+
+
+class TestAutoRoundCmd(unittest.TestCase):
+    @classmethod
+    def setUpClass(self):
+        pass
+
+    @classmethod
+    def tearDownClass(self):
+        shutil.rmtree("./saved", ignore_errors=True)
```

Comment:
defualt --output_dir is ./tmp_autoround
need shutil.rmtree("./tmp_autoround", ignore_errors=True)